testing/spectest/shared/deneb/ssz_static/ssz_static.go

```diff
@@ -47,12 +49,14 @@ func unmarshalledSSZ(t *testing.T, serializedBytes []byte, folderName string) (i
 	case "AggregateAndProof":
 		obj = &ethpb.AggregateAttestationAndProof{}
 	case "BeaconBlock":
+		t.Skip("Skipping BeaconBlock test")
```

Comment:
are we supposed to be skipping these tests? or is it taken cared of in a later pr

Reply:
fixed in 9f2d53754588ce54b10e297e8c765246964e069c